SECURITY.md

```diff
@@ -1,9 +1,38 @@
 # Security Policy
 
+This document outlines security procedures and general policies for the `OnDemand`
+project.
+
 ## Security Audits
 
-[Trusted CI](https://trustedci.org/) , the NSF Cybersecurity Center of Excellence conducted an in-depth vulnerability assessment of Open OnDemand, completing it in December 2018. This assessment included a careful review of the code, increasing our confidence in its security. The Ohio Supercomputing Center addressed the implementation issues (bugs) that were found during this review, producing a more robust revision of Open OnDemand.
+We have used [Trusted CI](https://trustedci.org/) in the past, the NSF Cybersecurity Center of 
+Excellence conducted an in-depth vulnerability assessment of Open OnDemand, completing 
+it in December 2018. This assessment included a careful review of the code, increasing 
+our confidence in its security. The Ohio Supercomputing Center addressed the implementation 
+issues (bugs) that were found during this review, producing a more robust revision of Open OnDemand.
+
+We hope to have more audits on a regular ongoing basis and to share those results when applicable.
 
 ## Reporting a Vulnerability
 
-Per our project website: http://openondemand.org/, if you have security concerns or think you have found a vulnerability in Open OnDemand, please contact us directly via email on the news list.  You can access the news list at https://lists.osu.edu/mailman/listinfo/ood-users or directly via ood-users@lists.osc.edu.  Emails sent to it are only seen by the core project team.
+
+If you have security concerns or think you have found a vulnerability in Open OnDemand,
+please contact us directly via [email](mailto:ood-users@lists.osc.edu) on the news list found [here](https://lists.osu.edu/mailman/listinfo/ood-users).
+
+Your email will be answered promptly by one of our team.
```

Comment:
Add back this sentence:

>Emails sent to it are only seen by the core project team.

That explains the privacy which is important. After adding that back this sentence becomes redundant and can be removed:

>Your email will be answered promptly by one of our team.

Reply:
Done.